mizani/transforms.py

```diff
@@ -588,10 +588,13 @@ def inverse(x):
                      doc=doc)
 
 
-logit_trans = probability_trans('logistic', _name='logit',
-                                _doc='Logit Transformation')
-probit_trans = probability_trans('norm', _name='norm',
-                                 _doc='Probit Transformation')
+try:
+    logit_trans = probability_trans('logistic', _name='logit',
+                                    _doc='Logit Transformation')
+    probit_trans = probability_trans('norm', _name='norm',
+                                     _doc='Probit Transformation')
+except ImportError:
+    pass
```

Comment:
More problems. Since `logit_trans` and `probit_trans` are included in `__all__`, they have to be defined, yet this could leave them undefined.!

Reply:
Oh yeah, good catch! `from mizani.transforms import *` crashes on this PR.

I guess that could be fixed by setting them both to `None`, or by only including them in `__all__` if they successfully get defined? Dunno which if either seems better.

(Or instead of `None`, setting them to strings like "Please install scipy...", or to functions that throw errors themselves, with docstrings like that? Dunno if that's better.)

Reply:
I think it may be easier to have `scipy` as a dependency, after all `mizani` is probably only used with `plotnine` which requires `scipy`.

Reply:
Yeah, seems reasonable. I'll close this.